ls1intum/Artemis
# `Exam mode`: Show correct working time for student exams with shorter working time

### Checklist
#### General


#### Client

### Motivation and Context


Fix a bug where exams with shorter working times (for example some test runs)show a wrong working time.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student

1. Create an exam in Artemis
2. Create a test run and a student exam that have a shorter working time than the normal working time of the exam
3. Check that the coversheet displays the correct end time and working time for both

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


Exam with normal working time and even additional working time presented in brackets:
![Bildschirmfoto 2021-10-11 um 22 57 40](https://user-images.githubusercontent.com/38322605/136854802-802d42d1-58fe-4b78-8dc3-5bbb8775287a.png)
Exam with a shorter working time (The settings in the exam itself are the same):
![Bildschirmfoto 2021-10-11 um 22 57 33](https://user-images.githubusercontent.com/38322605/136854847-d70c6e18-ba2e-478c-85df-0fdc186ba068.png)
